<!DOCTYPE html><!-- This site was created in Webflow. https://webflow.com --><!-- Last Published: Tue Oct 08 2024 07:49:19 GMT+0000 (Coordinated Universal Time) --><html data-wf-domain="yunmings-awesome-site.webflow.io" data-wf-page="66ff8d895370f19dc0047bdb" data-wf-site="66ff8d895370f19dc0047aeb" data-wf-status="1" lang="en"><head><meta charset="utf-8"><title>Styleguide</title><meta content="Styleguide" property="og:title"><meta content="Styleguide" property="twitter:title"><meta content="width=device-width, initial-scale=1" name="viewport"><meta content="Webflow" name="generator"><link href="static/css/yunmings-awesome-site.webflow.d182a5004.css" rel="stylesheet" type="text/css"><link href="https://fonts.googleapis.com/" rel="preconnect"><link href="https://fonts.gstatic.com/" rel="preconnect" crossorigin="anonymous"><script src="static/js/webfont.js" type="text/javascript"></script><script type="text/javascript">WebFont.load({  google: {    families: ["Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"]  }});</script><script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script><link href="https://cdn.prod.website-files.com/img/favicon.ico" rel="shortcut icon" type="image/x-icon"><link href="https://cdn.prod.website-files.com/img/webclip.png" rel="apple-touch-icon"></head><body><div data-animation="default" data-collapse="medium" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="navbar w-nav"><section class="section-2"><a href="index.html" class="w-nav-brand"><img sizes="120px" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-500.png 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-800.png 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-1080.png 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo.png 1150w" alt="" src="static/picture/66ff8e81ac60a3335ee21ae4_logo.png" loading="lazy" class="image"></a><nav role="navigation" class="nav-menu w-nav-menu"><a href="index.html" class="nav-link w-nav-link">Home</a><a href="project-design.html" class="nav-link w-nav-link">project design</a><a href="" aria-current="page" class="nav-link w-nav-link w--current">experiment methods</a><a href="results.html" class="nav-link w-nav-link">results</a><a href="team.html" class="nav-link w-nav-link">team</a></nav><div class="menu-button-2 w-nav-button"><div class="w-icon-nav-menu"></div></div></section></div><div class="div-block-4"><img sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac4f3bbfc5774f1b6133_20210403094444-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac4f3bbfc5774f1b6133_20210403094444-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac4f3bbfc5774f1b6133_20210403094444-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac4f3bbfc5774f1b6133_20210403094444-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac4f3bbfc5774f1b6133_20210403094444.jpg 1700w" alt="" src="static/picture/6704ac4f3bbfc5774f1b6133_20210403094444.jpg" loading="lazy" class="image-2"></div><div class="div-block-7"><div class="div-block-5"><h1 class="heading">Experiment Methods</h1><div class="text-cotent"><h3 class="heading-3"><strong>METHODS AND MATERIALS</strong></h3><h3 class="heading-5"><strong>Cell lines</strong></h3><p class="paragraph">The HEK-293 T, Expi293F, andJurkat cell lines utilized in this study were maintained within our laboratory.Jurkat cells were obtained from the Chinese Academy of Sciences (Shanghai,China) cell bank. All cell lines underwent STR authentication and routine qPCRtesting for mycoplasma contamination. DMEM supplemented with 10% fetal bovineserum (Gibco, USA), 100U/mL penicillin, and 100 mg/mL streptomycin (Gibco, USA)was used to culture all cells. </p></div><div class="text-cotent"><h3 class="heading-5"><strong>Construction of Gas6-Fc plasmid</strong></h3><p class="paragraph">The cDNAs of human Gas6 and Fc were obtained fromthe National Center for Biotechnology Information (NCBI) and amplified bypolymerase chain reaction (PCR). The functional domain of the Gas6 protein wasretrieved from UniProt. The Gas6-Fc fusion protein was optimized and generatedin collaboration with Sangon. The pcDNA3.1 vector underwent double digestion at37°C for 2 hours using restriction enzymes EcoR I and BamH I. Subsequently, thedigested vector was subjected to agarose gel electrophoresis analysis forverification. Finally, the digested vector was recovered using a DNA gelextraction kit. The PCR product was combined with the enzyme-digested vector ata specific ratio, followed by ligation using a homologous recombination enzyme.Subsequently, the ligation product was transformed into competent cells andplated for culture, wherein well-grown single clones were selected forsequencing verification. Upon confirming the accuracy of the sequence,large-scale culturing was conducted to extract plasmids, ultimately resultingin the recombinant plasmid harboring the target fusion protein genepcDNA3.1-Gas6-Fc.</p><h3 class="heading-5"><strong>Westernblot assays</strong></h3><p class="paragraph">The harvested cells were lysed in RIPA buffer, andthe protein concentrations were determined using BCA protein assay kits(Beyotime, Shanghai, China). Subsequently, the total protein lysates wereseparated by 12% sodium dodecyl sulfate polyacrylamide gel electrophoresis andtransferred onto PVDF membranes (Millipore, USA). Following a 1-hour blockingstep in Tris-buffered saline containing 5% non-fat milk and 0.5% BSA, themembranes were incubated overnight at 4°C with primary antibodies. Afterward,horseradish peroxidase (HRP)-conjugated secondary antibodies were applied for aduration of 1.5 hours at room temperature. Finally, chemiluminescent HRPsubstrate (Millipore, USA) was used to visualize the blots.</p><h3 class="heading-5"><strong>Purification and affinity analysis of Gas6-Fc fusion protein</strong></h3><p class="paragraph">1) Transfect pcDNA3.1-Gas6-Fcinto Expi293F cells expressing the target protein. The day prior totransfection, dilute Expi293F cells to a concentration of 2×106 cells/ml andculture them in a shaker at 37℃. On the day of transfection, dilute Expi293F cellsto a concentration of 5 × 106 cells/ml in 20 ml of fresh medium. Prepare thetransfection mixture by combining 2 mL T-pei buffer, 40μL of thepcDNA3.1-Gas6-Fc plasmid, and 120μL T-pei transfection reagent. Incubate thismixture for 15 minutes at room temperature. Gently add the mixture to the cellsuspension and continue culturing in the shaker at 37℃. After incubating for 24hours, supplement the culture with a combination of 3% (v/v) enhancer solutionand an enhancer agent (5‰). Three days post-transfection, centrifuge theculture to separate the supernatant containing expressed protein from cellulardebris. Resuspend the cells in fresh medium containing 3% 293 FA and 5‰enhancer, and return them to the shaker. The resulting supernatant can beimmediately subjected to protein purification using a protein G gel column orstored at -80°C with protease inhibitor for future use. On the sixth day,repeat the centrifugation step to collect additional supernatant. Optionally,retain a small portion of the cell pellet for verification of proteinexpression through techniques such as Western Blot (WB). If the target proteinis detected in either the supernatant or cell lysate, proceed with proteinpurification steps by passing the supernatant through a protein G gel column. </p><p class="paragraph">2) Purify the Gas6-Fc protein using protein G gelcolumn affinity chromatography. The supernatant was diluted with an equalvolume of Binding/Wash Buffer. Subsequently, the Protein G media was thoroughlymixed and packed into the chromatography column. The column was thenequilibrated with Binding/Wash Buffer to eliminate any air bubbles and looselybound media particles. The diluted sample was slowly introduced into the columnat a flow rate ranging from 0.2-0.5 mL/min. To enhance purification yield, multiplesample loadings could be performed if necessary. Following this step,approximately 30 mL of Binding/Wash Buffer was used to wash the column at aflow rate of about 2 mL/min in order to remove non-specifically boundimpurities. After washing, an Elution Buffer (10-15 mL) was added to the columnat a lower flow rate of approximately 1 mL/min for eluting the target protein,and subsequently collected as eluate. To achieve a pH of 7.4, the eluate wassupplemented with a neutralizing buffer in an amount equal to 1/10 of theeluate volume. Subsequently, the concentrations of the sample, flow-through,and eluate were quantified using a concentration detector. The purity of theeluate was confirmed by Coomassie Blue staining to ensure the presence of asingle target band. Once purity was verified, the protein sample containingonly one target band was treated with a protease inhibitor and temporarilystored at -80°C.</p><h3 class="heading-5"><strong>Flow cytometry</strong></h3><p class="paragraph">The cells (1 × 10<sup>6</sup>)were fixed in 4% formaldehyde for 15 minutes at room temperature. Afterwashing, the cells were incubated with primary antibodies for 1 hour and thenwith fluorescent secondary antibodies for 30 minutes at room temperature.Finally, the cells were analyzed using BD LSRFortessa Flow cytometry (BDBiosciences), and the data were analyzed using FlowJo software version 10(TreeStar, Inc.).</p><h3 class="heading-5"><strong>Real-time RT-PCR assays</strong></h3><p class="paragraph">The total RNA was extracted from macrophagescultured under normoxia or hypoxia for 12 h using an RNA-quick purification kit(Vazyme, RC102-01). Subsequently, the total RNA was reverse transcribed intocDNA utilizing the HiScript II 1st strand cDNA synthesis kit (Takara, RR036A).Quantitative real-time PCR analysis was conducted employing SYBR qPCR mastermix (Vazyme, Q711-02) and Applied Biosystems 7500 Fast instrument (LifeTechnologies). The primers utilized<br>for amplifying the targetgenes are presented as follows: <br>Mouse IL-6-F: TCCAGTTGCCTTCTTGGGAC; <br>Mouse IL-6-R: GTGTAATTAAGCCTCCGACTTGTG; <br>Mouse IL10-F: CAGGGATCTTAGCTAACGGAAA; <br>Mouse IL10-R: GCTCAGTGAATAAATAGAATGGGAAC; <br>Mouse actin-F: CTCTGTGTGGATCGGTGGC; <br>Mouse actin-R: GTAAAACGCAGCTCAGTAACAGTC. </p></div><div class="text-cotent"><h3 class="heading-5"><strong>Error-pronePCR</strong></h3><p class="paragraph">GoTaq polymerase was utilized in accordance with thecorresponding protocol, and mutation rates were calculated using theThermofisher fidelity calculator. To predict a single point mutation across theGla sequence, standard conditions were sufficient. Annealing temperatures wereadjusted to the primers, and PCR amplification was performed for 35 cycles. Theresulting PCR product was purified as per standard procedures.</p><h3 class="heading-5"><strong>Bacteriophages show Gla mutant library and pan sieve for high-affinitymutants</strong></h3><p class="paragraph">The reagent configurationcommonly employed for phage display experiments is as follows: 2×YT Medium:Dissolve 17 g of Bacto-tryptone, 10 g of Bacto-yeast extract, and 5 g of NaClin 900 ml of distilled water with stirring. Adjust the volume to 1 liter usingdistilled water and sterilize by autoclaving. SOBAG Medium: Autoclave a mixturecontaining 20 g of Bacto-tryptone, 5 g of Bacto-yeast extract, and 0.5 g ofNaCl in approximately 900 ml of distilled water. Once cooled to a temperaturebetween 50-60°C, add the following components: sterile solution (10 ml)containing MgCl2 at a concentration of 1 M, sterile solution (55.6 ml)containing glucose at a concentration of 2 M, and filter-sterilized ampicillin(5 ml) at a concentration of 20 mg/ml. For agar plates preparation, include anadditional amount of Bacto-agar (15 g) before autoclaving the medium. Pour theplates promptly after preparation. SB Medium: Dissolve 35 g of Bacto-tryptone,20 g of Bacto-yeast extract, and 5 g of NaCl in 900 ml of distilled water. Stiruntil completely dissolved. Adjust the pH to 7.5 with NaOH and make up thevolume to 1 liter with distilled water. Sterilize by autoclaving. TSS Buffer: Dissolve1.0 g of Bacto-tryptone, 0.5 g of Bactoyeast extract, and 0.5 g of NaCl in 70ml sterile distilled water. Subsequently, add 10.0 g polyethylene glycol (PEG,M.W.3350), followed by the addition of 5 ml dimethylsulfoxide (DMSO). Finally,incorporate a mixture containing either HCl or NaOH to adjust the pH toapproximately 6.5 and adjust the final volume to 100 ml using sterile distilledwater. Filter sterilize through a filter with a pore size around o.o22μm. Storeat a temperature as low as 4°C. The buffer can remain stable for up to sixmonths. PEG/NaCl: To prepare PEG/NaCl solution, dissolve 200 g Polyethyleneglycol 8000 (PEG) and146.1g NaCl in distilled water up to a total volume of oneliter. Heat until dissolved and autoclave it afterwards.Prepare1× TES Buffer bycombining Tris-HCl (pH8.O) (final concentration is 0.2M), EDTA (final concentrationis 0.5 mM) and sucrose (final concentration is 0.5M). Filter sterilize thissolution and store it at 4°C. To prepare l/5× TES Buffer, dilute l part ofl×TES Buffer with4 parts distilled water.</p><h3 class="heading-5"><strong>Statistical analysis</strong></h3><p class="paragraph">Statistical analyses were conducted using GraphPadPrism version 8.0 (GraphPad Software Inc.). All data are presented asmean ± SD. Student’s t tests with Welch correction were employed to analyzestatistical differences between two groups. One-way or two-way ANOVA with Sidakcorrection was used to assess statistical differences among three or moregroups. In all statistical analyses, significance was determined at the levelof **P &lt; 0.01, ***P &lt; 0.0001, ****P &lt; 0.01 and ns indicatednon-significance.</p></div></div></div><script src="static/js/jquery-3.5.1.min.dc5e7f18c8-66ff8d895370f19dc0047aeb.js" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script><script src="static/js/webflow.789cdef26.js" type="text/javascript"></script></body></html>