<!DOCTYPE html><!-- This site was created in Webflow. https://webflow.com --><!-- Last Published: Tue Oct 08 2024 07:49:19 GMT+0000 (Coordinated Universal Time) --><html data-wf-domain="yunmings-awesome-site.webflow.io" data-wf-page="66ff8d895370f19dc0047be3" data-wf-site="66ff8d895370f19dc0047aeb" data-wf-status="1" lang="en"><head><meta charset="utf-8"><title>Projects</title><meta content="Projects" property="og:title"><meta content="Projects" property="twitter:title"><meta content="width=device-width, initial-scale=1" name="viewport"><meta content="Webflow" name="generator"><link href="static/css/yunmings-awesome-site.webflow.d182a5004.css" rel="stylesheet" type="text/css"><link href="https://fonts.googleapis.com/" rel="preconnect"><link href="https://fonts.gstatic.com/" rel="preconnect" crossorigin="anonymous"><script src="static/js/webfont.js" type="text/javascript"></script><script type="text/javascript">WebFont.load({  google: {    families: ["Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"]  }});</script><script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script><link href="https://cdn.prod.website-files.com/img/favicon.ico" rel="shortcut icon" type="image/x-icon"><link href="https://cdn.prod.website-files.com/img/webclip.png" rel="apple-touch-icon"></head><body><div data-animation="default" data-collapse="medium" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="navbar w-nav"><section class="section-2"><a href="index.html" class="w-nav-brand"><img sizes="120px" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-500.png 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-800.png 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo-p-1080.png 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff8e81ac60a3335ee21ae4_logo.png 1150w" alt="" src="static/picture/66ff8e81ac60a3335ee21ae4_logo.png" loading="lazy" class="image"></a><nav role="navigation" class="nav-menu w-nav-menu"><a href="index.html" class="nav-link w-nav-link">Home</a><a href="project-design.html" class="nav-link w-nav-link">project design</a><a href="methods.html" class="nav-link w-nav-link">experiment methods</a><a href="" aria-current="page" class="nav-link w-nav-link w--current">results</a><a href="team.html" class="nav-link w-nav-link">team</a></nav><div class="menu-button-2 w-nav-button"><div class="w-icon-nav-menu"></div></div></section></div><div class="div-block-4"><img sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg%402o-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg%402o-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg%402o-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg%402o-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg%402o.jpg 1920w" alt="" src="static/picture/6704ac18dfd6ec2f1554fed8_01fbce5cc277a7a801214168bbf7d8.jpg@2o.jpg" loading="lazy" class="image-2"></div><div class="div-block-7"><div class="div-block-5"><h1 class="heading">Results</h1><div><h3 class="heading-3"><strong>Developa Gas6-Fc Fusion Protein to Ameliorate the Immunosuppressive TumorMicroenvironment through Directed Evolution Strategy</strong></h3><p>Shaoteng Lu, Haofeng Yin, Xibo Ye, YuchenGuo, Siqi Yang, Jiachen Zheng, Siyi Wang, Qianshan Hou, Chunzhen Li, Sheng Xu, TianliangLi, Shulei Yin, Yizhi YuNKLII-Evolution-ChinaNational Key Laboratory of Immunity andInflammation, Navy Medical University</p></div><div class="text-cotent"><h3 class="heading-4"><strong>Abstract</strong></h3><p class="paragraph"><p class="paragraph-2">The efferocytosis of tumor cells by tumor-associated macrophages (TAMs) leads to an immunosuppressive environment and tumor immune escape. Gas6 is a bridge protein between macrophages and tumor cells. How to modify Gas6 to change the immunosuppressive environment needs to be further explored. In this study, we have developed a novel recombinant fusion protein named Gas6-Fc, which retains the N-terminal domain of Gas6 for binding to apoptotic cells while substituting the C-terminal domain responsible for MerTK receptor binding with an antibody Fc fragment. By engaging with FcR instead of MerTK receptors on macrophages, Gas6-Fc promotes macrophage activation and triggers antibody-dependent cellular phagocytosis (ADCP) towards apoptotic cells. We aim to establish a eukaryotic expression system for the Gas6-Fc fusion protein and enhance its affinity towards tumor cells through directed evolution strategies, enabling it to effectively compete with endogenous Gas6. Subsequently, we will evaluate the impact of Gas6-Fc at both cellular and animal levels. The anticipated outcome is that the Gas6-Fc fusion protein will mitigate the immunosuppressive state within the tumor microenvironment and augment the efficacy of tumor therapies.</p></div><div class="text-cotent"><h3><strong class="bold-text">Keywords:</strong></h3><p class="paragraph">Gas6-Fc,Efferocytosis, Macrophage, Cancer, Directed evolution.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>Introduction</strong></h3><p class="paragraph">Phagocytes, such asmacrophages, play a crucial role in the recognition and engulfment of apoptoticcells, a process known as &#x27;<a href="#">efferocytosis</a>&#x27;[1]. Thisprocess is essential for maintaining tissue homeostasis under normalphysiological conditions and facilitating the restoration of homeostasisfollowing disease[2]. Efferocytosis efficiently removes apoptotic cells toprevent their secondary necrosis, which could lead to the release ofpotentially toxic intracellular contents and subsequent inflammation[3,4]. Moreover,phagocytes involved in efferocytosis undergo phenotypic changes towards a morepro-resolution state by increasing the secretion of anti-inflammatory mediatorsand upregulating pro-repair transcriptional programs[4,5]. This remarkablemechanism allows for the removal of billions of dying cells from the bodywithout inducing inflammation or immune responses. In contrast toefferocytosis, other forms of phagocytosis such as Fc receptor-mediatedphagocytosis are primarily aimed at initiating an immune response.<br><br>Theimmunologically silent process of efferocytosis, however, creates anenvironment for cancer cells to evade immune surveillance and thus promotestumor progression[6,7]. Harsh conditions within the tumor microenvironment,such as starvation, hypoxia, and chemotherapy drugs, induce apoptosis in tumorcells and trigger robust efferocytosis by tumor-associated macrophages.Following efferocytosis, tumor-associated macrophages upregulate the productionof anti-inflammatory cytokines, such as Transforming growth factor (TGF)β andinterleukin (IL)-10[7,8]. Additionally, tumor-associated macrophages expressvarious immune checkpoint ligands including programmed death-ligand 1 (PD-L1),leading to cytotoxic CD8 T cell starvation through arginase expression anddepletion of essential amino acids[9,10]. Furthermore, tumor-associatedmacrophages recruit regulatory T cells (Tregs), which contribute to theinhibition of antitumor immune responses by suppressing pro-inflammatorycytokine production and effector T cell function[11,12]. In this context, theaccumulation of tumor-associated macrophages is associated with an unfavorableprognosis in several cancer types including melanoma, breast, pancreatic, liver,ovarian, head and neck, bladder and renal cell cancer[10,13,14]. <br><br>Growtharrest-specific 6 (Gas6), a 75kDa vitamin K-dependent secreted protein, servesas a critical regulator of efferocytosis by acting as bridging proteins[15].The N-terminal GLA domain (γ-carboxyglutamate domain) of Gas6 specificallybinds to phosphatidylserine (PS) located on the surface of apoptotic cells,while the C-terminal region interacts with the MerTK receptor present onmacrophage surfaces[16]. This bridging effect facilitates the recognition,phagocytosis, and clearance of apoptotic cells by macrophages and inducesimmunosuppression in macrophages[15,17]. In the tumor microenvironment, thesustained activation of GAS6/MerTK signaling pathway leads to the generation ofa series of immunosuppressive signals that inhibit the host&#x27;s anti-tumor immuneresponse[18]. Consequently, this pathway mediates tumor drug resistance andimmune evasion, making it one of the most promising next-generation therapeutictargets for tumors[19–21]. <br><br>TheFc fusion protein refers to a genetically engineered protein that combines abiologically active protein segment with the Fc fragment in immunoglobulin[22].This fusion not only retains the specific function of the original protein butalso incorporates the functional characteristics of the Fc domain, making it avaluable therapeutic tool for various diseases, including cancer[23]. Comparedto the original protein, the Fc fusion protein exhibits an extended half-lifeand enhanced stability due to disulfide bond formation within its dimericstructure facilitated by the Fc domain[24,25]. Additionally, the Fc domain canmediate tumor-killing effects such as inflammation, antibody-dependent cellcytotoxicity (ADCC), and antibody-dependent cell phagocytosis (ADCP)[26,27]. Inthis study, we have successfully engineered, expressed, and purified theGas6-Fc fusion protein with the aim of enhancing the immunosuppressive tumormicroenvironment. In vitro experiments were conducted to validate thefunctionality of the fusion protein in specifically targeting apoptotic cellsand improving the immunosuppressive tumor microenvironment. Directed evolutionhas been employed to enhance the affinity of the fusion protein towardsapoptotic cell targets. </p></div><div class="text-cotent"><h3 class="heading-3"><strong>Results</strong></h3><h3 class="heading-5"><strong>Design strategies for Gas6-Fc fusion protein </strong></h3><p class="paragraph">Gas6 comprises the γ-carboxyglutamicacid (Gla) domain, four epidermal growth factor (EGF)-like domains, and twolaminin G-like (LG) domains (<strong>Fig. 1A</strong>). Considering the immunosuppressivefunction of Gas6 and the potent immune activation function of Fc, we retainedthe Gla domain and the EGF-like domain, which bind Ca<sup>2+</sup> to fulfillthe Ca<sup>2+</sup> dependency of Gla functioning, and fused them with the Fcfragment using a flexible linker (<strong>Fig. 1B</strong>).  Utilizing this protein design approach, wegenerated a eukaryotic expression plasmid for Gas6-Fc by optimizing pcDNA3.1vector with a flag tag integration (<strong>Fig. 1C</strong>). The gene sequence encodinghGas6 (GenBank accession no. NM_008202.4) was modified to retain its signalpeptide as well as Gla and EGF regions while also being tailored to enhancehost preference based on sequence alterations illustrated in <strong>Fig. 1D</strong>. Wepropose that linking Gas6-Fc between macrophages and tumor cells could triggeractivation signals while alleviating immunosuppression (<strong>Fig. 1E</strong>). Thewild-type Gas6 binds to apoptotic cells and initiates immunosuppressivesignals, leading to the polarization of macrophages towards the M2 phenotype,secretion of anti-inflammatory factors such as IL-10, and induction of immunetolerance. The N-terminus of Gas6-Fc competes with the wild-type Gas6 forbinding to PS on the surface of apoptotic cells. On the other hand, theC-terminus of Gas6-Fc triggers immune activation signals, polarizes macrophagestowards the M1 phenotype, and stimulates production of pro-inflammatory factorsincluding IL-1β, IL-18, and IFNβ.</p><img src="static/picture/66ff9564fa1e19beccb5badc_Figure1.jpg" loading="lazy" sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1-p-2000.jpg 2000w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff9564fa1e19beccb5badc_Figure1.jpg 2481w" alt=""><p class="paragraph"><strong>Figure 1. Thedesign strategy of the Gas6-Fc fusion protein.</strong> <strong>A</strong>The structures of Gas6. <strong>B</strong> The structures of Gas6-Fc. <strong>C</strong> Theplasmid map of Gas6-Fc. <strong>D</strong> Comparative display of sequence changes. <strong>E</strong>Schematic diagrams of the interaction between apoptoticcell/Gas6-Fc/macrophage.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>Preparation of Gas6-Fc fusion protein</strong></h3><p class="paragraph">The pcDNA3.1-Gas6-Fc-Flagplasmid was transfected into HEK-293T cells for 24 hours, and the supernatantand cell lysate from the transfected cells were collected. The proteins in thesupernatant were enriched through anti-Flag immunoprecipitation. In comparisonto the untransfected sample, both the enriched supernatant and cell lysatesamples from HEK-293T cells transfected with Gas6-Fc plasmid exhibited adistinct band at a molecular weight of 63 KD, which corresponded to thepredicted size of Gas6-Fc (<strong>Fig. 2A</strong>). These results demonstratesuccessful expression of the Gas6-Fc plasmid.<br><br>In order to minimize theimpact of serum on isolation and purification (and to reduce costs), weutilized Expi293F cells, which can be cultured in a serum-free medium forlarge-scale protein production, to achieve high-level expression of the Gas6-Fcfusion protein. The Gas6-Fc plasmid was transiently transfected into Expi239Fcells. Subsequently, Gas6-Fc in the cell culture media was purified usingProtein G affinity chromatography (<strong>Fig. 2B</strong>).  Protein samples at various purification stageswere analyzed by 12% SDS-PAGE. Coomassie brilliant blue staining resultsindicated successful protein expression and purification in Expi239F cells (<strong>Fig.2C</strong>). The results demonstrated successful expression of Gas6-Fc fusionprotein in the culture media of Expi293F cells, followed by its effectiveisolation through protein G affinity chromatography. Furthermore, we performedan additional column repetition to enhance the efficiency of fusion proteinenrichment, resulting in a final yield of 5ml fusion protein at a concentrationof 128ug/ml.</p><img src="static/picture/66ff96877c909f6051345b92_Figure2.jpg" loading="lazy" sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2-p-2000.jpg 2000w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96877c909f6051345b92_Figure2.jpg 2481w" alt=""><p class="paragraph"><strong>Figure 2. Preparationprocedure for the fusion protein Gas6-Fc.</strong> <strong>A </strong>Protein expression insupernatant and cell lysate of HEK-293T after transfection with Gas6-Fc plasmidfor 24 h. <strong>B</strong> Flow chart of Gas6-Fc protein purified by protein G affinitychromatography. <strong>C</strong> SDS-PAGE identification of purified Gas6-Fc protein.M: marker; 1: purified Gas6-Fc; 2: eluted effluent; 3: original solution; 4:after a purification of the original solution.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>Functional validation of Gas6-Fc fusionproteins—targeting apoptotic cells</strong></h3><p class="paragraph">The ability of theengineered Gas6-Fc fusion protein to specifically target phosphatidylserine(PS) on the surfaces of apoptotic cells was confirmed through flow cytometryanalysis. Human T-lymphocytic leukemia cells (Jurkat) were treated with staurosporine(STS) at a ratio of 250:1 for a duration of 3 hours to induce apoptosis.AnnexinV-FITC, a PS binding protein, was used to detect the percent ofapoptosis cells. After treatment with STS, the percentage of apoptotic Jurkatcells significantly increased from 3.5% to 98.1% (<strong>Fig. 3A, B</strong>).Subsequently, apoptotic Jurkat cells were incubated with Gas6-Fc or isotypeIgG. The binding of Gas6-Fc to apoptotic cell surfaces was detected using ananti-Fc (PE) secondary antibody. As expected, it was found that IgG did notbind to Jurkat cells either with or without apoptotic treatment (<strong>Fig. 3C, D</strong>).In contrast, The binding ratio of Ga6 to Jurkat cells exhibited a significantincrease following apoptosis induction, rising from 13.3% to 58.6% (<strong>Fig. 3C,D</strong>). These findings strongly suggest that the Gas6-Fc fusion proteinexhibits an affinity for PS on the surface of apoptotic cells and caneffectively target them.</p><img src="static/picture/66ff96f793c3e52de30e08f9_Figure3.jpg" loading="lazy" sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3-p-2000.jpg 2000w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff96f793c3e52de30e08f9_Figure3.jpg 2481w" alt=""><p class="paragraph"><strong>Figure 3. Thetargeting capability of GAS6-Fc towards apoptotic cells.</strong> <strong>A </strong>Flow cytometryquantification of apoptotic Jurkat cells with or withoutSTS treatment. <strong>B</strong> Rate of apoptotic Jurkat cells with or withoutSTS treatment. <strong>C</strong> The binding rate of Gas6-Fc to Jurkat cells wasquantified using flow cytometry, with detection by an anti-Fc (PE) antibody. <strong>D</strong>Positive rate of Jurkat cells binding to Gas6-Fc.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>Gas6-Fc fusion protein relieved the immunesuppressive</strong></h3><p class="paragraph">In order to validate the functionalityof Gas6-Fc fusion protein in inducing immune activation of macrophages, wecultured macrophages with Gas6-Fc fusion protein in a culture medium containingapoptotic Jurkat cells. The presence of apoptotic cells would induce theefferocytosis of macrophages. After a 7-hour incubation period, the RNA ofmacrophages was extracted and cytokine expression was assessed using qPCR. Thelevels of immune-activating cytokines, including IL6 and IFN-β,increased significantly with 100ug/ml Gas6-Fc administration duringefferocytosis compared to the control group without Gas6-Fc treatment (<strong>Fig.4A</strong>), indicating Gas6-Fc can enhance the secretion of pro-inflammatorycytokines. In contrast, administration of Gas6-Fc resulted in a decrease in theexpression level of IL10, an immunosuppressive cytokine, in macrophagescompared to the control group (<strong>Fig. 4B</strong>). 1. Overall, these data suggestthat Gas6-Fc can shift macrophages from an immunosuppressive state to an immune-activatingstate during efferocytosis.</p><img src="static/picture/66ff974e052ef82c86caa5b6_Figure4.jpg" loading="lazy" sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4-p-2000.jpg 2000w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff974e052ef82c86caa5b6_Figure4.jpg 2481w" alt=""><p class="paragraph"><strong>Figure 4. The Gas6-Fc fusion proteinalleviates immune suppression of macrophages.A</strong> The RNA levels of IL6 and IFN-β in macrophages treated with Gas6-Fcfusion protein. <strong>B</strong> The RNA level of IL10 in macrophages treated withGas6-Fc fusion protein. 1 μg/ml, 10 μg/ml, and 100 μg/ml are the threeconcentration gradients of Gas6-Fc fusion protein in the system. The datarepresent three independent replicate experiments.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>The Gla and EGF domains presented as PIIIfusion proteins on filamentous phages.</strong></h3><p class="paragraph">Considering that only ahigh concentration (100ug/ml) of Gas6-Fc exhibited an ameliorated immunesuppression effect, and taking into account the <em>in vivo</em> experimentaldosage as well as clinical drug requirements, our objective is to furtherinvestigate methods for optimizing the fusion protein&#x27;s affinity to PS on apoptoticcells. As, the Gla and EGF domains of Gas6 are crucial regions for binding toPS, we introduced random mutations into these two regions by error-prone PCR. Inspiredby the epitope localization function of phage display technology, weestablished a phage peptide display library targeting the mutated Gla and EGFfragments (<strong>Fig. 5A</strong>). The phage display procedure was as follows: Phagedisplay library generation: The gene fragments with random mutations wereobtained using error-prone PCR, followed by enzymatic ligation for theirinsertion into phasmid vectors. The library phasmids were then transferred intoE. coli via electroshocking to establish a bacterial library. Helper phage wasintroduced to invade the E. coli cells, facilitating the amplification of phagewithin the bacterium and ultimately constructing a recombinant phage librarycapable of displaying mutated Gla-EGF fragment protein on its surface. (2)Targeted binding: Gla-EGF fragment protein-display phages were incubated with magneticnanoparticles loaded with PS. (3) Washing: We eliminated the phages thatexhibit non-binding or weak binding interactions with PS-loaded nanoparticles,and specifically isolate the phages with high affinity. (4) Phage Elution: Thehigh affinity phage was eluted for subsequent sequence analysis. (5) Phagere-amplification: Take a small amount of phage to reinfect E.coli and introducehelper phage for proliferation, in preparation for another enrichment. Afterseveral cycles, collect purified phagemids for analysis. Randomly mutated Gla-EGFfragments were generated using the GeneMorphII random mutagenesis kit. The fragmentswere inserted between the SacI and SpeI enzyme cleavage sites adjacent to the5&#x27; end of M13 gene PIII, resulting in a genetically engineered constructsuitable for displaying the corresponding protein fragments (<strong>Fig. 5B</strong>). Thedisplayed protein in our system is tagged with an HA tag, and its recognitionserves as confirmation for the presence of each individual Gla-EGF fragment onthe phage surface (<strong>Fig. 5B</strong>). The analysis of the double digestionproduct of the phagemid vector pcomb3xss using 1% agarose gel electrophoresisrevealed two distinct bands that corresponded to the expected size (3300 bp +1600 bp) (<strong>Fig. 5C</strong>). The gel electrophoresis analysis revealed thepresence of mutated Gla-EGF fragments generated through error-prone PCR, withan expected band size ranging between 500 and 750bp (<strong>Fig. 5D</strong>). Thephages enriched with a high affinity for PS have been submitted for sequencing,pending subsequent functional validation of the high-affinity mutant. Moreover,the antitumor function of the evolved Gas6-Fc would be evaluated through an <em>invivo</em> tumor model,</p><img src="static/picture/66ff98533e1ce13089bcfd6f_Figure5.jpg" loading="lazy" sizes="100vw" srcset="https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5-p-500.jpg 500w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5-p-800.jpg 800w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5-p-1080.jpg 1080w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5-p-1600.jpg 1600w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5-p-2000.jpg 2000w, https://cdn.prod.website-files.com/66ff8d895370f19dc0047aeb/66ff98533e1ce13089bcfd6f_Figure5.jpg 2481w" alt=""><p class="paragraph"><strong>Figure 5. Strategy of affinity directed evolution.</strong> <strong>A</strong> Schematic diagram of phagedisplay technology <strong>B</strong> Schematic representation of phagemid vector. <strong>C</strong> Phagemidvector pComb3xss was digested by double enzyme (spe1/sac1) and detected usinggel electrophoresis <strong>D</strong> The generated Gla-EGF fragments by error-prone PCRwere detected using <a href="#">gel electrophoresis</a>.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>Discussion</strong></h3><p class="paragraph">Incontrast to wild-type Gas6, the absence of the MerTK binding domain in Gas6-Fcprevents M2 polarization of macrophages. Instead, through its Fc domain bindingto FcγRs, Gas6-Fc promotes M1 polarization by activating the NF-κB signalingpathway and enhancing secretion of IFN-β, IL-18, and IL-1β by macrophages. Consequently,this activation leads to enhanced anti-tumor immune responses. Moreover,Gas6-Fc also activates diverse immune cells involved in antibody-dependentcell-mediated cytotoxicity (ADCC), antibody-dependent cellular phagocytosis(ADCP), and complement-dependent cytotoxicity (CDC) effects, thereby furtheraugmenting tumor cell clearance. Engineered through directed evolution, Gas6-Fcexhibits significantly higher affinity for PS than wild-type Gas6 and isexpected to competitively inhibit the binding between wild-type Gas6 andapoptotic cells at low concentrations. While wild-type Gas6 has a smallermolecular weight and is susceptible to hydrolysis by endogenous proteases andpeptidases, Gas6-Fc possesses greater protein stability.<br>‍<br>Inorder to enhance the affinity of Gas6-Fc for PS, we introduced random mutationsin a specific region of Gas6 using error-prone PCR to establish a diverselibrary of Gas6 mutants. To enrich for mutants with the highest affinity forPS, we employed phage display technology, known for its cost-effectiveness,high efficiency, and short cycles. The peptide library was displayed on thesurface of M13 phage and incubated with PS-coated magnetic beads duringselection. Subsequently, we enriched for Gas6 mutants exhibiting the strongestbinding affinity towards PS. Furthermore, our future studies will investigatethe potential influence of the Fc segment on the interaction between the Gas6segment and PS. <br><br>Inaddition to Gas 6, there exist other PS-binding proteins that can be engineeredinto Fc-fusion proteins to exert therapeutic effects. For instance, Annexin A5is a calcium-dependent phospholipid-binding protein exhibiting high affinitytowards PS... It has been demonstrated that the interaction between Annexin A5and PS on apoptotic cell surfaces can modulate PS-mediated immune suppression,augment the immunogenicity of apoptotic cells, and alleviate immune suppressiveconditions within the tumor microenvironment. Due to its smaller size andnon-secretory nature, Annexin A5&#x27;s relatively shorter half-life may impacttreatment efficacy or necessitate higher dosages. Consequently, fusion ofAnnexin A5 with the Fc region could potentially prolong its half-life andenhance therapeutic effectiveness.</p></div><div class="text-cotent"><h3 class="heading-5"><strong>References</strong></h3><p class="paragraph">1    Boada-RomeroE, Martinez J, Heckmann BL, <em>et al.</em> The clearance of dead cells byefferocytosis. <em>Nat Rev Mol Cell Biol</em>. 2020;21:398–414. doi:10.1038/s41580-020-0232-12    <br>2、GerlachBD, Ampomah PB, Yurdagul A, <em>et al.</em> Efferocytosis induces macrophageproliferation to help resolve tissue injury. <em>Cell Metab</em>.2021;33:2445-2463.e8. doi: 10.1016/j.cmet.2021.10.0153    <br>3、DoranAC, Yurdagul A, Tabas I. Efferocytosis in health and disease. <em>Nat RevImmunol</em>. 2020;20:254–67. doi: 10.1038/s41577-019-0240-64    <br>4、PoonIKH, Ravichandran KS. Targeting Efferocytosis in Inflammaging. <em>Annu RevPharmacol Toxicol</em>. 2024;64:339–57. doi:10.1146/annurev-pharmtox-032723-1105075    <br>5、MehrotraP, Ravichandran KS. Drugging the efferocytosis process: concepts andopportunities. <em>Nat Rev Drug Discov</em>. 2022;21:601–20. doi:10.1038/s41573-022-00470-y6    <br>6、LecoultreM, Dutoit V, Walker PR. Phagocytic function of tumor-associated macrophages asa key determinant of tumor progression control: a review. <em>Journal forImmunotherapy of Cancer</em>. 2020;8:e001408. doi: 10.1136/jitc-2020-0014087    <br>7、ZhouY, Yao Y, Deng Y, <em>et al.</em> Regulation of efferocytosis as a novel cancertherapy. <em>Cell communication and signaling: CCS</em>. 2020;18:71. doi:10.1186/s12964-020-00542-98    <br>8、QiuH, Shao Z, Wen X, <em>et al.</em> Efferocytosis: An accomplice of cancer immuneescape. <em>Biomedicine &amp; Pharmacotherapy</em>. 2023;167:115540. doi:10.1016/j.biopha.2023.1155409    <br>9、ZhangH, Liu L, Liu J, <em>et al.</em> Roles of tumor-associated macrophages inanti-PD-1/PD-L1 immunotherapy for solid cancers. <em>Mol Cancer</em>. 2023;22:58.doi: 10.1186/s12943-023-01725-x10  <br>10、XiangX, Wang J, Lu D, <em>et al.</em> Targeting tumor-associated macrophages tosynergize tumor immunotherapy. <em>Signal Transduction and Targeted Therapy</em>.2021;6:75. doi: 10.1038/s41392-021-00484-911  <br>11、LiC, Xu X, Wei S, <em>et al.</em> Tumor-associated macrophages: potentialtherapeutic strategies and future prospects in cancer. <em>Journal forImmunotherapy of Cancer</em>. 2021;9:e001341. doi: 10.1136/jitc-2020-00134112  <br>12、PathriaP, Louis TL, Varner JA. Targeting Tumor-Associated Macrophages in Cancer. <em>TrendsImmunol</em>. 2019;40:310–27. doi: 10.1016/j.it.2019.02.00313  <br>13、KumariN, Choi SH. Tumor-associated macrophages in cancer: recent advancements incancer nanoimmunotherapies. <em>Journal of experimental &amp; clinical cancerresearch: CR</em>. 2022;41:68. doi: 10.1186/s13046-022-02272-x14  <br>14、Cheng K, Cai N, Zhu J, <em>et al.</em> Tumor-associatedmacrophages in liver cancer: From mechanisms to therapy. <em>CancerCommunications</em>. 2022;42:1112–40. doi: 10.1002/cac2.1234515  <br>15、WuG, Ma Z, Cheng Y, <em>et al.</em> Targeting Gas6/TAM in cancer cells and tumormicroenvironment. <em>Mol Cancer</em>. 2018;17:20. doi: 10.1186/s12943-018-0769-116  <br>16、WuG, Ma Z, Hu W, <em>et al.</em> Molecular insights of Gas6/TAM in cancerdevelopment and therapy. <em>Cell Death Dis</em>. 2017;8:e2700. doi:10.1038/cddis.2017.11317  <br>17、VagoJP, Amaral FA, van de Loo FAJ. Resolving inflammation by TAM receptoractivation. <em>Pharmacol Ther</em>. 2021;227:107893. doi:10.1016/j.pharmthera.2021.10789318  <br>18、MyersKV, Amend SR, Pienta KJ. Targeting Tyro3, Axl and MerTK (TAM receptors):implications for macrophages in the tumor microenvironment. <em>Mol Cancer</em>.2019;18:94. doi: 10.1186/s12943-019-1022-219  <br>19、Adam-ArtiguesA, Arenas EJ, Martínez-Sabadell A, <em>et al.</em> Targeting HER2-AXLheterodimerization to overcome resistance to HER2 blockade in breast cancer. <em>SciAdv</em>. 2022;8:eabk2746. doi: 10.1126/sciadv.abk274620  <br>20、Apostolo D, Ferreira LL, Vincenzi F, <em>et al.</em>From MASH to HCC: the role of Gas6/TAMreceptors. <em>Front Immunol</em>. 2024;15:1332818. doi:10.3389/fimmu.2024.133281821  <br>21、ZhuC, Wei Y, Wei X. AXL receptor tyrosine kinase as a promising anti-cancerapproach: functions, molecular mechanisms and clinical applications. <em>MolCancer</em>. 2019;18:153. doi: 10.1186/s12943-019-1090-322  <br>22、RashidMH. Full-length recombinant antibodies from Escherichia coli: production,characterization, effector function (Fc) engineering, and clinical evaluation. <em>MAbs</em>. 2022;14:2111748. doi: 10.1080/19420862.2022.211174823  <br>23、Niu Y-X, Xu Z-X, Yu L-F, <em>et al.</em> Advancesof research of Fc-fusion protein that activate NK cells for tumorimmunotherapy. <em>Int Immunopharmacol</em>. 2022;109:108783. doi:10.1016/j.intimp.2022.10878324  <br>24、RamdaniY, Lamamy J, Watier H, <em>et al.</em> Monoclonal Antibody Engineering and Designto Modulate FcRn Activities: A Comprehensive Review. <em>Int J Mol Sci</em>.2022;23:9604. doi: 10.3390/ijms2317960425  <br>25、FrommG, de Silva S, Schreiber TH. Reconciling intrinsic properties of activating TNFreceptors by native ligands versus synthetic agonists. <em>Front Immunol</em>.2023;14:1236332. doi: 10.3389/fimmu.2023.123633226  <br>26、YuJ, Song Y, Tian W. How to select IgG subclasses in developing anti-tumortherapeutic antibodies. <em>J Hematol Oncol</em>. 2020;13:45. doi:10.1186/s13045-020-00876-427   <br>27、Musolino A, Gradishar WJ, Rugo HS, <em>et al.</em>Role of Fcγ receptors in HER2-targeted breast cancer therapy. <em>J ImmunotherCancer</em>. 2022;10:e003171. doi: 10.1136/jitc-2021-003171</p></div><div class="text-cotent"><h3 class="heading-3"><strong>METHODS AND MATERIALS</strong></h3><h3 class="heading-5"><strong>Cell lines</strong></h3><p class="paragraph">The HEK-293 T, Expi293F, andJurkat cell lines utilized in this study were maintained within our laboratory.Jurkat cells were obtained from the Chinese Academy of Sciences (Shanghai,China) cell bank. All cell lines underwent STR authentication and routine qPCRtesting for mycoplasma contamination. DMEM supplemented with 10% fetal bovineserum (Gibco, USA), 100U/mL penicillin, and 100 mg/mL streptomycin (Gibco, USA)was used to culture all cells. </p></div><div class="text-cotent"><h3 class="heading-5"><strong>Construction of Gas6-Fc plasmid</strong></h3><p class="paragraph">The cDNAs of human Gas6 and Fc were obtained fromthe National Center for Biotechnology Information (NCBI) and amplified bypolymerase chain reaction (PCR). The functional domain of the Gas6 protein wasretrieved from UniProt. The Gas6-Fc fusion protein was optimized and generatedin collaboration with Sangon. The pcDNA3.1 vector underwent double digestion at37°C for 2 hours using restriction enzymes EcoR I and BamH I. Subsequently, thedigested vector was subjected to agarose gel electrophoresis analysis forverification. Finally, the digested vector was recovered using a DNA gelextraction kit. The PCR product was combined with the enzyme-digested vector ata specific ratio, followed by ligation using a homologous recombination enzyme.Subsequently, the ligation product was transformed into competent cells andplated for culture, wherein well-grown single clones were selected forsequencing verification. Upon confirming the accuracy of the sequence,large-scale culturing was conducted to extract plasmids, ultimately resultingin the recombinant plasmid harboring the target fusion protein genepcDNA3.1-Gas6-Fc.</p><h3 class="heading-5"><strong>Westernblot assays</strong></h3><p class="paragraph">The harvested cells were lysed in RIPA buffer, andthe protein concentrations were determined using BCA protein assay kits(Beyotime, Shanghai, China). Subsequently, the total protein lysates wereseparated by 12% sodium dodecyl sulfate polyacrylamide gel electrophoresis andtransferred onto PVDF membranes (Millipore, USA). Following a 1-hour blockingstep in Tris-buffered saline containing 5% non-fat milk and 0.5% BSA, themembranes were incubated overnight at 4°C with primary antibodies. Afterward,horseradish peroxidase (HRP)-conjugated secondary antibodies were applied for aduration of 1.5 hours at room temperature. Finally, chemiluminescent HRPsubstrate (Millipore, USA) was used to visualize the blots.</p><h3 class="heading-5"><strong>Purification and affinity analysis of Gas6-Fc fusion protein</strong></h3><p class="paragraph">1) Transfect pcDNA3.1-Gas6-Fcinto Expi293F cells expressing the target protein. The day prior totransfection, dilute Expi293F cells to a concentration of 2×106 cells/ml andculture them in a shaker at 37℃. On the day of transfection, dilute Expi293F cellsto a concentration of 5 × 106 cells/ml in 20 ml of fresh medium. Prepare thetransfection mixture by combining 2 mL T-pei buffer, 40μL of thepcDNA3.1-Gas6-Fc plasmid, and 120μL T-pei transfection reagent. Incubate thismixture for 15 minutes at room temperature. Gently add the mixture to the cellsuspension and continue culturing in the shaker at 37℃. After incubating for 24hours, supplement the culture with a combination of 3% (v/v) enhancer solutionand an enhancer agent (5‰). Three days post-transfection, centrifuge theculture to separate the supernatant containing expressed protein from cellulardebris. Resuspend the cells in fresh medium containing 3% 293 FA and 5‰enhancer, and return them to the shaker. The resulting supernatant can beimmediately subjected to protein purification using a protein G gel column orstored at -80°C with protease inhibitor for future use. On the sixth day,repeat the centrifugation step to collect additional supernatant. Optionally,retain a small portion of the cell pellet for verification of proteinexpression through techniques such as Western Blot (WB). If the target proteinis detected in either the supernatant or cell lysate, proceed with proteinpurification steps by passing the supernatant through a protein G gel column. </p><p class="paragraph">2) Purify the Gas6-Fc protein using protein G gelcolumn affinity chromatography. The supernatant was diluted with an equalvolume of Binding/Wash Buffer. Subsequently, the Protein G media was thoroughlymixed and packed into the chromatography column. The column was thenequilibrated with Binding/Wash Buffer to eliminate any air bubbles and looselybound media particles. The diluted sample was slowly introduced into the columnat a flow rate ranging from 0.2-0.5 mL/min. To enhance purification yield, multiplesample loadings could be performed if necessary. Following this step,approximately 30 mL of Binding/Wash Buffer was used to wash the column at aflow rate of about 2 mL/min in order to remove non-specifically boundimpurities. After washing, an Elution Buffer (10-15 mL) was added to the columnat a lower flow rate of approximately 1 mL/min for eluting the target protein,and subsequently collected as eluate. To achieve a pH of 7.4, the eluate wassupplemented with a neutralizing buffer in an amount equal to 1/10 of theeluate volume. Subsequently, the concentrations of the sample, flow-through,and eluate were quantified using a concentration detector. The purity of theeluate was confirmed by Coomassie Blue staining to ensure the presence of asingle target band. Once purity was verified, the protein sample containingonly one target band was treated with a protease inhibitor and temporarilystored at -80°C.</p><h3 class="heading-5"><strong>Flow cytometry</strong></h3><p class="paragraph">The cells (1 × 10<sup>6</sup>)were fixed in 4% formaldehyde for 15 minutes at room temperature. Afterwashing, the cells were incubated with primary antibodies for 1 hour and thenwith fluorescent secondary antibodies for 30 minutes at room temperature.Finally, the cells were analyzed using BD LSRFortessa Flow cytometry (BDBiosciences), and the data were analyzed using FlowJo software version 10(TreeStar, Inc.).</p><h3 class="heading-5"><strong>Real-time RT-PCR assays</strong></h3><p class="paragraph">The total RNA was extracted from macrophagescultured under normoxia or hypoxia for 12 h using an RNA-quick purification kit(Vazyme, RC102-01). Subsequently, the total RNA was reverse transcribed intocDNA utilizing the HiScript II 1st strand cDNA synthesis kit (Takara, RR036A).Quantitative real-time PCR analysis was conducted employing SYBR qPCR mastermix (Vazyme, Q711-02) and Applied Biosystems 7500 Fast instrument (LifeTechnologies). The primers utilized<br>for amplifying the targetgenes are presented as follows: <br>Mouse IL-6-F: TCCAGTTGCCTTCTTGGGAC; <br>Mouse IL-6-R: GTGTAATTAAGCCTCCGACTTGTG; <br>Mouse IL10-F: CAGGGATCTTAGCTAACGGAAA; <br>Mouse IL10-R: GCTCAGTGAATAAATAGAATGGGAAC; <br>Mouse actin-F: CTCTGTGTGGATCGGTGGC; <br>Mouse actin-R: GTAAAACGCAGCTCAGTAACAGTC. </p></div><div class="text-cotent"><h3 class="heading-5"><strong>Error-pronePCR</strong></h3><p class="paragraph">GoTaq polymerase was utilized in accordance with thecorresponding protocol, and mutation rates were calculated using theThermofisher fidelity calculator. To predict a single point mutation across theGla sequence, standard conditions were sufficient. Annealing temperatures wereadjusted to the primers, and PCR amplification was performed for 35 cycles. Theresulting PCR product was purified as per standard procedures.</p><h3 class="heading-5"><strong>Bacteriophages show Gla mutant library and pan sieve for high-affinitymutants</strong></h3><p class="paragraph">The reagent configurationcommonly employed for phage display experiments is as follows: 2×YT Medium:Dissolve 17 g of Bacto-tryptone, 10 g of Bacto-yeast extract, and 5 g of NaClin 900 ml of distilled water with stirring. Adjust the volume to 1 liter usingdistilled water and sterilize by autoclaving. SOBAG Medium: Autoclave a mixturecontaining 20 g of Bacto-tryptone, 5 g of Bacto-yeast extract, and 0.5 g ofNaCl in approximately 900 ml of distilled water. Once cooled to a temperaturebetween 50-60°C, add the following components: sterile solution (10 ml)containing MgCl2 at a concentration of 1 M, sterile solution (55.6 ml)containing glucose at a concentration of 2 M, and filter-sterilized ampicillin(5 ml) at a concentration of 20 mg/ml. For agar plates preparation, include anadditional amount of Bacto-agar (15 g) before autoclaving the medium. Pour theplates promptly after preparation. SB Medium: Dissolve 35 g of Bacto-tryptone,20 g of Bacto-yeast extract, and 5 g of NaCl in 900 ml of distilled water. Stiruntil completely dissolved. Adjust the pH to 7.5 with NaOH and make up thevolume to 1 liter with distilled water. Sterilize by autoclaving. TSS Buffer: Dissolve1.0 g of Bacto-tryptone, 0.5 g of Bactoyeast extract, and 0.5 g of NaCl in 70ml sterile distilled water. Subsequently, add 10.0 g polyethylene glycol (PEG,M.W.3350), followed by the addition of 5 ml dimethylsulfoxide (DMSO). Finally,incorporate a mixture containing either HCl or NaOH to adjust the pH toapproximately 6.5 and adjust the final volume to 100 ml using sterile distilledwater. Filter sterilize through a filter with a pore size around o.o22μm. Storeat a temperature as low as 4°C. The buffer can remain stable for up to sixmonths. PEG/NaCl: To prepare PEG/NaCl solution, dissolve 200 g Polyethyleneglycol 8000 (PEG) and146.1g NaCl in distilled water up to a total volume of oneliter. Heat until dissolved and autoclave it afterwards.Prepare1× TES Buffer bycombining Tris-HCl (pH8.O) (final concentration is 0.2M), EDTA (final concentrationis 0.5 mM) and sucrose (final concentration is 0.5M). Filter sterilize thissolution and store it at 4°C. To prepare l/5× TES Buffer, dilute l part ofl×TES Buffer with4 parts distilled water.</p><h3 class="heading-5"><strong>Statistical analysis</strong></h3><p class="paragraph">Statistical analyses were conducted using GraphPadPrism version 8.0 (GraphPad Software Inc.). All data are presented asmean ± SD. Student’s t tests with Welch correction were employed to analyzestatistical differences between two groups. One-way or two-way ANOVA with Sidakcorrection was used to assess statistical differences among three or moregroups. In all statistical analyses, significance was determined at the levelof **P &lt; 0.01, ***P &lt; 0.0001, ****P &lt; 0.01 and ns indicatednon-significance.</p></div></div></div><script src="static/js/jquery-3.5.1.min.dc5e7f18c8-66ff8d895370f19dc0047aeb.js" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script><script src="static/js/webflow.789cdef26.js" type="text/javascript"></script></body></html>